man/man1/pdfsizeopt.1

```diff
@@ -0,0 +1,185 @@
+.TP
+\fB--do-unify-pages=\fRYES_NO
+default: \fIyes\fR
+  Unify duplicate /Type /Page objects? It makes the PDF with duplicate pages a bit smaller. Disabling it increases compatibility with some old PDF viewers.
+.TP
+\fB--do-generate-xref-stream=\fRYES_NO
+default: \fIyes\fR
+  Generate the output PDF file with an xref stream (rather than an xref)? It produces smaller output. It should only be turned off for debugging.
+.TP
+\fB--do-generate-object-stream=\fRYES_NO
+default: \fIyes\fR
+  Generate the output PDF file with non-stream objects packed to an object stream (objstm)?
+  It produces smaller output. It should only be turned off for debugging.
+.TP
+\fB--do-decompress-flate=\fRYES_NO
+default: \fIno\fR
+  Decompress all /Filter /FlateEncode streams in the output PDF? Doing so can make the output PDf much larger, so it should be enabled only for debugging, like this:
+  \fI--do-decompress-flate=yes --do-compress-uncompressed-streams=no\fR
+.TP
+\fB--do-decompress-most-streams=\fRYES_NO
+default: \fIno\fR
+  Decompress most compressed streams in the output PDF? Doing so can make the output PDf much larger, so it should be enabled only for debugging, like this:
+  \fI--do-decompress-most-streams=yes --do-compress-uncompressed-streams=no\fR
+  Actually not all streams are decompressed, but only those which eventually get used in the output PDF, except if they have filter /DCTDecode or /JPXDecode (because such decompression is not deterministic and fully accurate).
+.TP
+\fB--do-compress-uncompressed-streams=\fRYES_NO
+default: \fIyes\fR
+  Compress stream objects in the output which are uncompressed? This is applied after the decompressions by the \fI--do-decompres-...\fR flags.
+.TP
+\fB--use-multivalent=\fRYES_NO
+default: \fIno\fR
+  Use the tool.pdf.Compress tool in Multivalent.jar to optimize the PDF even further, after applying the optimizations built in to pdfsizeopt? Not enabled by default, because it needs a copy of Multivalent.jar, which is not available for download or buying anymore.
+.TP
+\fB--do-escape-images-from-multivalent=\fRYES_NO
+default: \fIyes\fR
+  Hide images from Multivalent.jar, making it skip all image optimizations?
+  It's useful to enable it by default, because Multivalent has some image processing bugs (including some compression predictor handling), thus it would garble the image with a /Predictor.
+.TP
+\fB--do-debug-gs=\fRYES_NO
+default: \fIno\fR
+  Display debug info about where pdfsizeopt is trying to find Ghostscript, whether the found Ghostscripts work, and which one was chosen? All this before reading the first input PDF.
+
+.SH EXAMPLE
+\fBpdfsizeopt\fR sample.pdf
+
+.SH COPYRIGHT
+Copyright 2007 Péter Szabó.
+ All rights reserved.
+
+.SH SEE ALSO
+ pdfinfo(1), pngout(1), jpdftweak(1).
```

Comment:
jpdftweak is not relevant, please remove it. Please add jbig2(1), sam2p(1), imgdataopt(1). gs(1) and qpdf(1).